test/test_nn.py

```diff
@@ -3453,7 +3453,7 @@ def _test_GroupNorm_general(self, device="cpu", dtype=torch.float):
             self.assertRaises(RuntimeError, lambda: gn(input))
 
     def _test_GroupNorm_cuda_half(self):
-        input = Variable(torch.empty(2, 3, 3, 2).to("cuda", torch.half).random_(1, 10), requires_grad=True)
+        input = torch.empty(2, 3, 3, 2, requires_grad=True).to("cuda", torch.half).random_(1, 10)
```

Comment:
Could we just remove this line? `input` is already being overwritten by the very next line.

Reply:
Go for it. It doesn't look significant.